### ISSUE TITLE
OpenGraphParser URI encoding?

### ISSUE THREAD

**Issue by apparition47** [Osedea/react-native-opengraph-kit#17]
If I enter:
```
https://column.clintal.com/健診を受ける/がん検診/7012
```
Your example app incorrectly displays the Open Graph data of the root webpage (`https://column.clintal.com` in this case).

![simulator screen shot - iphone 8 - 2018-02-28 at 14 45 32](https://user-images.githubusercontent.com/3298414/36771857-42b9e5b8-1c96-11e8-8619-d3c5ab27bbfd.png)

If I manually encode the non-latin characters:
```
https://column.clintal.com/%E5%81%A5%E8%A8%BA%E3%82%92%E5%8F%97%E3%81%91%E3%82%8B/%E3%81%8C%E3%82%93%E6%A4%9C%E8%A8%BA/7012
```
The example app displays the correct Open Graph data.

![simulator screen shot - iphone 8 - 2018-02-28 at 14 45 18](https://user-images.githubusercontent.com/3298414/36771867-47bbe872-1c96-11e8-8308-8eed602d5be9.png)

I think running the URL through [`encodeURI()`](https://www.w3schools.com/jsref/jsref_encodeURI.asp) first will solve the issue on the parser side.

Thoughts?

**Comment by apparition47** [comment]
Turns out the root of the issue isn't so much related to URL encoding as it is the regex used to match URLs in a given message. Created a PR with a regex that works in my case and should work in other cases with other international URLs.

**Comment by adrienthiery** [comment]
Solved by #18

**Comment by adrienthiery** [comment]
And published 1.6.1 right now 🙂
